integrations/nlu-rasa/src/RasaNlu.ts

```diff
@@ -49,4 +96,14 @@ export class RasaNlu extends NluPlugin<RasaNluConfig> {
   ): Promise<AxiosResponse<RasaResponse>> {
     return axios.post(`${this.config.serverUrl}${this.config.serverPath}`, { text }, config);
   }
+
+  private mapAlternativeIntents(altIntents: RasaIntent[]) {
```

Comment:
A return-type is required to satisfy eslint.

Reply:
added return type